src/Utilities/Compiler/Options/AnalyzerOptionsExtensions.cs

```diff
@@ -128,57 +128,57 @@ static bool TryParseValue(string value, out ImmutableHashSet<TEnum> result)
             return analyzerConfigOptions.GetOptionValue(optionName, rule, uint.TryParse, defaultValue);
         }
 
-        public static SymbolNamesOption GetNullCheckValidationMethodsOption(
+        public static SymbolNamesOption<byte> GetNullCheckValidationMethodsOption(
```

Comment:
I am not a big fan of the `<byte>` but I couldn't find anything nice except for adding a `Unit` class.